### PR TITLE
feat: adding Curve pools: PAX, hBTC, metapools: gUSD, hUSD, USDn, mUSD, tBTC

### PR DESCRIPTION
**Pull request by romainbutteaud**
## Description
- Curve Pools: PAX, hBTC
- Curve Metapools: USD, hUSD, USDn, mUSD, tBTC
- Simbot: https://metabase.spaceship.0x.org/dashboard/139?run_id=new-curve-pools-01

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [x] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/asset-swapper/src/utils/market_operation_utils/constants.ts [hunk 1]
```diff
@@ -131,54 +161,98 @@ export const MAINNET_CURVE_INFOS: { [name: string]: CurveInfo } = {
...
+        poolAddress: POOLS.curve_TRI,
         tokens: [TOKENS.DAI, TOKENS.USDC, TOKENS.USDT],
     },
+    // Metapools
+    GUSD: {
+        exchangeFunctionSelector: CurveFunctionSelectors.exchange_underlying,
+        sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy_underlying,
+        buyQuoteFunctionSelector: CurveFunctionSelectors.None,
+        poolAddress: POOLS.curve_GUSD,
+        tokens: [TOKENS.GUSD, TOKENS.DAI, TOKENS.USDC, TOKENS.USDT],
```
Comment (dekz) [inline packages/asset-swapper/src/utils/market_operation_utils/constants.ts:226]: Do these pools clash with the TriPool? 

In the case of a DAI/USDC sell would the GUSD pool and the TriPool both return samples and potentially result in trading against the same underlying (TriPool)?

Or does it not work like that, and the trade has to include GUSD?

File: packages/asset-swapper/src/utils/market_operation_utils/constants.ts [hunk 2]
```diff
@@ -131,54 +161,98 @@ export const MAINNET_CURVE_INFOS: { [name: string]: CurveInfo } = {
...
-    TriPool: {
+    HBTC: {
         exchangeFunctionSelector: CurveFunctionSelectors.exchange,
         sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy,
         buyQuoteFunctionSelector: CurveFunctionSelectors.None,
-        poolAddress: '0xbebc44782c7db0a1a60cb6fe97d0b483032ff1c7',
+        poolAddress: POOLS.curve_HBTC,
+        tokens: [TOKENS.hBTC, TOKENS.WBTC],
+    },
+    TRI: {
```
Comment (dekz) [inline packages/asset-swapper/src/utils/market_operation_utils/constants.ts:213]: I dont think we actually use these names anywhere, so we may as well use the address as a name?

```suggestion
    [POOLS.curve_TRI]: {
```

File: packages/asset-swapper/src/utils/market_operation_utils/constants.ts [hunk 3]
```diff
@@ -361,19 +422,25 @@ export const DEFAULT_GAS_SCHEDULE: Required<FeeSchedule> = {
     [ERC20BridgeSource.Curve]: fillData => {
         const poolAddress = (fillData as CurveFillData).pool.poolAddress.toLowerCase();
         switch (poolAddress) {
-            case '0xa5407eae9ba41422680e2e00537571bcc53efbfd':
-            case '0x93054188d876f558f4a66b2ef1d97d16edf0895b':
-            case '0x7fc77b5c7614e1533320ea6ddc2eb61fa00a9714':
-            case '0xbebc44782c7db0a1a60cb6fe97d0b483032ff1c7':
+            case POOLS.curve_renBTC:
```
Comment (dekz) [inline packages/asset-swapper/src/utils/market_operation_utils/constants.ts:425]: Much betterer

File: packages/asset-swapper/src/utils/market_operation_utils/constants.ts (hunk 2 above)
Comment (romainbutteaud) [reply packages/asset-swapper/src/utils/market_operation_utils/constants.ts:213]: 👍  good point

File: packages/asset-swapper/src/utils/market_operation_utils/constants.ts (hunk 1 above)
Comment (romainbutteaud) [reply packages/asset-swapper/src/utils/market_operation_utils/constants.ts:226]: good point, let me try to run a few tests

**Comment by romainbutteaud** [comment]
Simbot: https://metabase.spaceship.0x.org/dashboard/139?run_id=protocol-pr26-001&adjusted_amount_bought_win_tolerance__bps_=0.5
Token list for this run (19): `ETH,IMBTC,EDC,YUSD,UNI,USDN,AMPL,RSR,SNX,DAI,HBTC,HUSD,MUSD,PAX,TBTC,USDT,USDC,WETH,WBTC`

File: packages/asset-swapper/src/utils/market_operation_utils/constants.ts (hunk 1 above)
Comment (romainbutteaud) [reply packages/asset-swapper/src/utils/market_operation_utils/constants.ts:235]: ✅

**Comment by romainbutteaud** [comment]
r4r @dekz: now filtering out metapools if the `makerToken` or the `takerToken` isn't the metatoken of the pool.

**Review by dekz — APPROVED** [review]
Need to bump up the change log off of dev. But lgtm!

**Pull request merged**